### PR TITLE
#4098 [Manager] Default value of Duration is not displayed in the edit conn…

### PR DESCRIPTION
**Pull request by konekoya**
…ector property dialog

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4098)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/Pipeline/PipelinePropertyDialog/PipelinePropertyDialog.js [hunk 1]
```diff
@@ -127,8 +127,37 @@ const PipelinePropertyDialog = props => {
...
+        const hasSeparator =
+          currentValue.length > 0 && currentValue.includes(' ');
+
+        const unit = 'milliseconds';
+
+        // Backend is always use `milliseconds`, so we need to do the conversion
+        // * 1000
+        values[valueKey] = hasSeparator
+          ? currentValue
+          : `${currentValue * 1000} ${unit}`;
```
Comment (eechih) [inline ohara-manager/client/src/components/Pipeline/PipelinePropertyDialog/PipelinePropertyDialog.js]: Can we move this logic to components/common/Definitions/Duration.js onChange()

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyDialog/PipelinePropertyDialog.js]: No problem, I will give a shot :)

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/PipelinePropertyDialog/PipelinePropertyDialog.js]: The result is pretty amazing ❤️ I have get rid of most of the logic of finding def keys. Thanks for the tip :)

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**